ui/raidboss/data/06-ew/raid/p4n.ts

```diff
@@ -0,0 +1,198 @@
+import NetRegexes from '../../../../../resources/netregexes';
+// import Outputs from '../../../../../resources/outputs';
+// import { callOverlayHandler } from '../../../../../resources/overlay_plugin_api';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { PluginCombatantState } from '../../../../../types/event';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  bodyActor?: PluginCombatantState;
+}
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.AsphodelosTheFourthCircle,
+  triggers: [
+    {
+      id: 'P4N Elegant Evisceration',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A50', source: 'Hesperos' }),
+      response: Responses.tankCleave(),
+    },
+    // Strong proximity Aoe
+    {
+      id: 'P4N Levinstrike Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A3F', source: 'Hesperos', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go to Corners',
+        },
+      },
+    },
+    // Knockback from middle
+    {
+      id: 'P4N Well Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A3E', source: 'Hesperos', capture: false }),
+      delaySeconds: 4,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'P4N Acid Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A3C', source: 'Hesperos', capture: false }),
+      response: Responses.spread(),
+    },
+    {
+      id: 'P4N Lava Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A3D', source: 'Hesperos', capture: false }),
+      response: Responses.stackMarker(),
+    },
+    {
+      id: 'P4N Decollation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A51', source: 'Hesperos', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P4N Bloodrake',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A40', source: 'Hesperos', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P4N Hell Skewer',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A4F', source: 'Hesperos', capture: false }),
+      response: Responses.awayFromFront(),
+    },
+    {
+      id: 'P4N Belone Coils',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '69DD', source: 'Hesperos', capture: false }),
+      delaySeconds: 2,
+      alarmText: (data, _matches, output) => {
```

Comment:
alarmText seems like a lot here.  Maybe infoText?

Reply:
Still unresolved.

Reply:
Oh sorry. I didn't mean to be rude I though this was about the text being too long so I checked these two off. I should take more time to read...

I changed both in 296760177934d1b86b7ea5a8749618ce8a354173

Reply:
I didn't take it as rude! Sorry if I came off as curt.  I figured it was either forgotten / accidentally not included / misunderstood or something along those lines.  I could have been more clear!